docker/docker-compose.1804.50.yaml

```diff
@@ -26,6 +26,10 @@ services:
       - MAX_ALLOCS_ALLOWED_scheduling_10000_executions=20150
       - MAX_ALLOCS_ALLOWED_creating_10000_headers=10100
       - MAX_ALLOCS_ALLOWED_modifying_1000_circular_buffer_elements=50
+      - MAX_ALLOCS_ALLOWED_encode_1000_ws_frames_holding_buffer=4010
+      - MAX_ALLOCS_ALLOWED_encode_1000_ws_frames_holding_buffer_with_space=4010
+      - MAX_ALLOCS_ALLOWED_encode_1000_ws_frames_new_buffer=6010
+      - MAX_ALLOCS_ALLOWED_encode_1000_ws_frames_new_buffer_with_space=6010
```

Comment:
How do you come up with these value, do you just run once and figure out whether the number is reasonable/expected i.e. X allocs per whatever-is-being measured?

Reply:
need to run it (on Linux) and seed with the initial numbers with what you see. Typically we add between 10 and 50 to the result to cover for any slack. And typically we run something at least 1000 times, so anything below 1000 is definitely slack. But these things are usually super stable, so 10 slack is usually good enough. I don't think we've ever seen a failure because of slack.

Reply:
So the idea is: Let's seed it with something we've seen. And from then on we know when we regress (and also if we improve).

Reply:
Yeah, the system is to first run them and see what the numbers are. If they're insane (they were the first time), find where the extra allocs have come from and kill them. Then, push to CI, let the CI tell you what the answers it got were, and put those in with a buffer of four or five allocations to allow for different Swift versions to impose new costs at startup.